tests/unit/common/standaloneInfoHelper.test.js

```diff
@@ -0,0 +1,11 @@
+import { generateClientDeviceID } from "../../../src/js/common/standaloneInfoHelper";
+
+describe("StandaloneInfoHelper", () => {
+    it("generateClientDevice", () => {
+        const generatedUUID = generateClientDeviceID();
+        expect(generatedUUID.split("-").length).toEqual(5);
+        expect(generatedUUID.length).toEqual(36);
+        expect(typeof generatedUUID).toBe("string");
```

Comment:
I think this test could be a bit more solid. While we don't want to test uuid itself (so adding a regex here probably isn't helpful) we do want to make sure that our function doesn't do anything funky to the output of uuid.

I would therefore stub uuid so `uuid()` returns a set UUID string. Then verify that the return from this function is the same string without any modifications.

If I went into your function and changed the code from 

```js
export const generateClientDeviceID = () => uuidV4(); 
```

to

```js
export const generateClientDeviceID = () => '123e4567-e89b-12d3-a456-426614174000'
```
or
```js
export const generateClientDeviceID = () => '123e4567e89b12d3a45642661417-4-0-0-0'
```

Then it wouldn't fail... but obviously it would be a bug we'd want to catch